### ISSUE TITLE
Sometimes spans do not follow the actual execution order

### ISSUE THREAD

**Issue by sergimola** [nioc/node-red-contrib-opentelemetry#10]
Sometimes spans do not follow the structure according to the actual execution of the flow.

Here's a simple example:
![image](https://github.com/user-attachments/assets/cc4e8b72-ac9b-4b51-804f-a101b942f3cc)

Correct one:
![image](https://github.com/user-attachments/assets/1c29f56a-73a4-45b1-bc58-709f7acbc60b)

Wrong one:
![image](https://github.com/user-attachments/assets/47e3669a-388c-44b6-8272-569d826adb63)

It doesn't happen only on this one, but it happens in general

**Comment by nioc** [comment]
Yes I already seen this behavior when operation is so fast that the nodejs engine hasn't been able to process the sending of the trace in the right order.
Unfortunately I don't think it can be fixed.

**Comment by nioc** [comment]
Hello @sergimola, with updated OTEL sdk, does this behavior is still present?

**Comment by nioc** [comment]
I close it, feel free to reopen if relevant

**Comment by sergimola** [comment]
Sadly it's still happening, but I understand fixing it would require quite some extra effor.
